pom.xml

```diff
@@ -435,6 +446,24 @@
                 <version>${gravitee-node.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>io.gravitee.node</groupId>
+                <artifactId>gravitee-node-reporter</artifactId>
+                <version>${gravitee-node.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.gravitee.node</groupId>
+                <artifactId>gravitee-node-monitoring</artifactId>
+                <version>${gravitee-node.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.gravitee.node</groupId>
+                <artifactId>gravitee-node-plugins-service</artifactId>
+                <version>${gravitee-node.version}</version>
+            </dependency>
+
```

Comment:
I think gravitee-node-xxx dependencies can be replaced with the following in the dependency management as it auto imports all sub-modules.

```
<dependency>
    <groupId>io.gravitee.node</groupId>
    <artifactId>gravitee-node</artifactId>
    <version>${gravitee-node.version}</version>
</dependency>
```